setup.py

```diff
@@ -60,7 +60,8 @@
       install_requires=[
             'numpy>=1.16',
             'pyspline>=1.1',
-            'scipy>=1.2'
+            'scipy>=1.2',
+            'numpy-stl'
```

Comment:
I was going to say that I needed to manually install this package to run the tests but I see you added them as a dependency. I did not need to re-run setup after pulling from your branch so not sure how we can fix this problem for other users who will just do a git pull and not do the setup again

Reply:
This only breaks tests if they don't have numpy-stl, not the package itself so I think it's OK

Reply:
IMO we shouldn't get all users to install this package by default. The proper way is to have `optional_dependencies` for testing, for example
```
optional_dependencies = {
    'testing': [
        'numpy-stl',
    ],
}
```
and you will install all the required plus the `testing` optional dependencies if you use `pip install . [testing]`. (Can someone test that this actually works, this is what OpenMDAO does but I don't know if install locally with `.` together)